tests/test_images.sh

```diff
@@ -251,8 +251,9 @@ setup()
 	[ ! -d "${tests_repo_dir}" ] && git clone "https://${tests_repo}" "${tests_repo_dir}"
 
 	if [ -z "${KATA_DEV_MODE:-}" ]; then
-		chronic $mgr remove-docker
-		chronic $mgr install-docker-system
+		"${tests_repo_dir}/.ci/setup.sh"
```

Comment:
there is also https://github.com/kata-containers/tests/blob/master/.ci/install_kata.sh

Reply:
It's worth pointing out here the subtle distinction: the `kata-manager.sh` script installs *packages* built from `master`, whereas this PR is allowing *components* to be built from `master` (at the point before they are packaged ;)

Reply:
@jcvenegas thanks, let me try with `install_kata.sh`

Reply:
@jcvenegas no, `install_kata.sh` didn't work because there are a lot of build dependencies